net/dnscrypt-proxy2/Makefile

```diff
@@ -25,6 +25,7 @@ PKG_BUILD_PARALLEL:=1
 PKG_USE_MIPS16:=0
 
 GO_PKG:=github.com/DNSCrypt/dnscrypt-proxy
+PKG_CONFIG_DEPENDS := CONFIG_$(PKG_NAME)_INCLUDE_GOPROXY
```

Comment:
Please do not use these confusing $(PKG_NAME) variables in symbol and sector declarations, unless absolutely necessary. They just confuse, and may prevent searching.